src/api/ratelimits.mdx

```diff
@@ -7,7 +7,7 @@ Sentry rate limits every API request made to prevent abuse and resource overuse.
 
 The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
 
-While there is a default value, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response.
+While there is a default value of 40 requests per second, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response.
```

Comment:
```suggestion
While there is a default rate limit of <b>40 requests per second</b>, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response.
```

I don't remember the syntax for bold text, lol

Reply:
lol I think it's asterisks, but I'll make this change.